opentelemetry-otlp/tests/integration_test/README.md

```diff
@@ -15,3 +15,42 @@ popped back out into the files output by the collector.
 * Docker, for the test container
 * TCP/4317 and TCP/4318 free on your local machine. If you are running another
   collector, you'll need to stop it for the tests to run.
+
+## Test Matrix
+
+| **Exporter Type**   | **Provider Creation**     | **Logging**           | **Network Clients**        | **Status** |
+|----------------------|---------------------------|------------------------|----------------------------|------------|
+| **Simple**           | Inside `tokio::Runtime`  | Inside `tokio::Runtime` | `tonic`                    | Works      |
```

Comment:
nit: It would be nice to use check marks here similar to the test matrix for metrics: https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/src/metrics/test.md

![image](https://github.com/user-attachments/assets/b33d72b2-1085-4d34-b10a-2eb4baf44197)


```suggestion
| **Simple**           | Inside `tokio::Runtime`  | Inside `tokio::Runtime` | `tonic`                    | :white_check_mark: |
```